### ISSUE TITLE
every number is s.whatsapp.net exists with contactsync.php

### ISSUE THREAD

**Issue by hablas** [mgp25/Chat-API#653]
sorry for lot of questions ... but  i want to use your great API  :+1: 
i try to sync number to me with this page : contactsync.php

$username = "xxxxxxxxx";
$password = "xxxxxxxxxxx";
$u        = "**************";    // i put here the number i want to sync

i try a lot of numbers , but all results is : **************@s.whatsapp.net exists

**Comment by mgp25** [comment]
Oks, then numbers exists.

**Comment by hablas** [comment]
sorry @mgp25 
i don't understand your answer .
i still have the problem
cant sync numbers

**Comment by mgp25** [comment]
What are you using the API for?

**Comment by hablas** [comment]
web service 
to send and received messages from my whats app

**Comment by mgp25** [comment]
@hablas do you know whatsapp web exists? https://web.whatsapp.com/ that service already exists. Im still curious about what usage will you use it.

**Comment by hablas** [comment]
i know this amazing service
but i make a small work for one of my clients
and the issue of this ticket is working now with me 
thank you very much

but i still have question :
how i know that the number is already added to my contact list ?

**Comment by hablas** [comment]
i see this page : https://github.com/mgp25/Chat-API/wiki/Sync
and know if the message : echo "$number exists<br />"; appear .
that mean its already added to my contact list 
thanks alot @mgp25  :+1:
